addons/mail/static/tests/helpers/model_definitions_setup.js

```diff
@@ -43,6 +43,7 @@ insertModelFields('mail.channel', {
             return this.currentPartnerId;
         },
     },
+    authorizedGroupFullName: { type: 'char' },
```

Comment:
This should not be here as fake field, it should be in mock server as return value.